provision/docker/provisioner.go

```diff
@@ -312,6 +312,9 @@ func (p *dockerProvisioner) Start(app provision.App, process string) error {
 		return errors.New(fmt.Sprintf("Got error while getting app containers: %s", err))
 	}
 	err = runInContainers(containers, func(c *container.Container, _ chan *container.Container) error {
+		if c.Status == provision.StatusStarted.String() {
```

Comment:
Can you add a test to this case?

Reply:
Also, I don't think we should blindly trust on the information provided by MongoDB as it might be stale, the start operation should still be issued to the containers and we could check the returned `startErr` and handle it accordingly.